<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eggsDialog</class>
 <widget class="QDialog" name="eggsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Enable eggs</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="enableEgg">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Enable binary notation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="binaryButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Enable image jokes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="imageJokes">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Begin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="beginTime"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTimeEdit" name="endTime"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>pressed()</signal>
   <receiver>eggsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>binaryButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageJokes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>beginTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>endTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableEgg</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
